File: Spring Server/src/main/java/com/app/MyCoolApp/models/Quiz.java
package com.app.MyCoolApp.models;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Table(name="Quiz")
@Data
public class Quiz {
    @Id
    @GeneratedValue(strategy= GenerationType.IDENTITY)
    @Column(name="id")
    private Integer id;

    @Column(name="quiztitle")
    private String quiztitle;

    @Column(name="quizdesc")
    private String quizdesc;

    @Column(name="creator_id")
    private String creator_id;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getQuiztitle() {
        return quiztitle;
    }

    public void setQuiztitle(String quiztitle) {
        this.quiztitle = quiztitle;
    }

    public String getQuizdesc() {
        return quizdesc;
    }

    public void setQuizdesc(String quizdesc) {
        this.quizdesc = quizdesc;
    }

    public String getCreator_id() {
        return creator_id;
    }

    public void setCreator_id(String creator_id) {
        this.creator_id = creator_id;
    }

    @Override
    public String toString() {
        return "Quiz{" +
                "id=" + id +
                ", quiztitle='" + quiztitle + '\'' +
                ", quizdesc='" + quizdesc + '\'' +
                ", creator_id='" + creator_id + '\'' +
                '}';
    }
}

File: androidquiz/lib/entities/quiz.dart
class Quiz {
  final int id;
  final String quiztitle;
  final String quizdesc;
  final int creator_id;

  String getQuiztitle(){
    return quiztitle;
  }
  String getQuizdesc(){
    return quizdesc;
  }
  int getCreator_id(){
    return creator_id;
  }
  int getQuizId(){
    return id;
  }

  const Quiz({
    required this.id,
    required this.quiztitle,
    required this.quizdesc,
    required this.creator_id,
  });
  
  factory Quiz.fromJson(Map<String, dynamic> json ){
    return Quiz(
      id: json['id'],
      quiztitle: json['quiztitle'],
      quizdesc: json['quizdesc'],
      creator_id: json['creator_id'],
    );
  }
}